### PR TITLE
Alerting: Make two boxes in template form adaptative to the screen 

### LINKED ISSUE #67574: Support adaptive widths for Content and Data textareas
**What**

It would be amazing if we could make the width of the Content and Data textareas adaptive so it fits on the same line in `1600x900`. We should also test `1920x1080` and `1366x768`.

Here with the menu expanded Data is pushed onto the next line:

Screenshot 2023-04-29 at 13 03 56

Here with the menu collapsed both textareas are on the same line:

Screenshot 2023-04-29 at 13 04 07

It would be awesome if we could make the textareas adaptive so both fit on the same line even with the menu expanded:

Screenshot 2023-04-29 at 13 03 56

### PR DESCRIPTION
**Pull request by soniaAguilarPeiron**
**What is this feature?**

This PR makes two editors in template form to be aligned on the same line when possible (fixed a minimum width of 450px so the editors don't become too tight).

**Why do we need this feature?**

Having two editors at the same line it's better for the user as it can compare information easily.

**Who is this feature for?**

All users.

**Which issue(s) does this PR fix?**:

Fixes https://github.com/grafana/grafana/issues/67574

**Special notes for your reviewer:**

**Before:**

[before1.webm](https://user-images.githubusercontent.com/33540275/236763498-ece597b3-253c-4ceb-bdf1-802ee1874691.webm)

**After:** 

[after5.webm](https://user-images.githubusercontent.com/33540275/236778699-ad96c82e-e809-4446-83c8-5a68e40e5c36.webm)

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/app/features/alerting/unified/components/receivers/TemplateForm.tsx [hunk 1]
```diff
@@ -378,8 +388,8 @@ export function TemplatePreview({
   useEffect(() => onPreview(), [onPreview]);
 
   return (
-    <Stack direction="row" alignItems="center" gap={2}>
-      <Stack direction="column">
+    <div style={{ width: `${width}px` }}>
```
Comment (konrad147) [inline public/app/features/alerting/unified/components/receivers/TemplateForm.tsx:391]: Why do we need two `div` containers here? Can't we achieve the same using one `div`?

File: public/app/features/alerting/unified/components/receivers/TemplateForm.tsx [hunk 2]
```diff
@@ -417,52 +434,49 @@ const getStyles = (theme: GrafanaTheme2) => ({
     font-weight: ${theme.typography.fontWeightBold};
   `,
   buttons: css`
+      display: flex;
     & > * + * {
-      margin-left: ${theme.spacing(1)};
+        margin - left: ${theme.spacing(1)};
```
Comment (konrad147) [inline public/app/features/alerting/unified/components/receivers/TemplateForm.tsx:439]: Invalid formatting?

Comment (soniaAguilarPeiron) [reply public/app/features/alerting/unified/components/receivers/TemplateForm.tsx:439]: oh 🤔

File: public/app/features/alerting/unified/components/receivers/TemplateForm.tsx (hunk 1 above)
Comment (soniaAguilarPeiron) [reply public/app/features/alerting/unified/components/receivers/TemplateForm.tsx]: good catch! solved and pushed! @konrad147

File: public/app/features/alerting/unified/components/receivers/TemplateForm.tsx (hunk 2 above)
Comment (soniaAguilarPeiron) [reply public/app/features/alerting/unified/components/receivers/TemplateForm.tsx]: fixed ✅

**Review by konrad147 — APPROVED** [review]
LGTM!

**Pull request merged**